v22.2/export.md

```diff
@@ -34,6 +34,20 @@ The `EXPORT` statement cannot be used within a [transaction](transactions.html).
 
  The user must have the `SELECT` [privilege](security-reference/authorization.html#managing-privileges) on the table being exported, unless the [destination URI requires `admin` privileges](import.html#source-privileges).
 
+### Destination privileges
+
+ The user that runs `EXPORT` to external storage does **not** require the `ADMIN` role in the following scenarios:
+
+- Amazon S3 and Google Cloud Storage using `SPECIFIED` credentials. Azure Storage is always `SPECIFIED` by default.
+- [Userfile](use-userfile-for-bulk-operations.html)
+
+{% include {{ page.version.version }}/misc/external-io-privilege.md %}
+
+The user that runs `EXPORT` to external storage **does** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
```

Comment:
Same as the comment in destination-privileges.md I don't know if we want to give the user the impression that there are still operations that only admin can run, because thats not the case anymore. EXTERNALIOIMPLICITACCESS governs who can and cannot run against an ExternalStorage.